namespaces/collectd.yml

```diff
@@ -1,39 +1,54 @@
-
 namespace:
   name: collectd
   type: group
   description: >
-    Namespace metrics metadata
+    Namespace metrics metadata.
   fields:
-  - name: vm_name
+  - name: interval
+    type: float
+    description: >
+      Collectd's interval.
+
+  - name: plugin
     type: string
     description: >
-      Virtual Machine name is in plugin_instance for virt plugin records.
+      Collectd's plugin.
```

Comment:
Its the same convention as the other descriptions, like interval, plugin_instance, type etc.
I don't think its critical. I can update if needed.